co-log/tutorials/2-custom/Custom.md

```diff
@@ -49,15 +49,15 @@ data Env m = Env
 
 Several notes about this data type:
 
-1. It stores different parameters, like server port.
-2. It stores `LogAction` that can log `Message` data type from `co-log` in the
-   `m` monad.
-3. `Env` is parameterized by type variable `m` which is going to be application
-   monad.
+1. It stores different parameters, like the server port.
+2. It stores a `LogAction` that can log `Message`s from `co-log` in the `m`
```

Comment:
I would keep "data type` as it shows better what `Message` we are talking about.
cc @chshersh

Reply:
I agree with @vrom911. Let's keep definintion of `Message` since this word is too generic. Alternatively, this part can be rewritten to:

```
... that can log values of the `Message` type from `co-log` ...
```